wsg_50_driver/src/test_finger_read.cpp

```diff
@@ -0,0 +1,34 @@
+#include <ros/ros.h>
+#include "wsg_50/hand.h"
+#include "wsg_50_common/Status.h"
+
+#include <iostream>
+#include <fstream>
+
+int main(int argc, char** argv) {
+
+    ros::init(argc, argv, "test_hand_read");
+
+    ros::NodeHandle nh;
+
+    Hand hand(nh, "hand0");
+    hand.start_reading();
+    hand.pause_hand_reading();
+    hand.pause_finger_reading(1);
+
+    ros::Duration(1.0).sleep();
+
+    ros::Rate loop_rate(30);
+    while(ros::ok()) {
+        wsg_50_common::WeissFingerData finger0_sample = hand.get_finger_sample(0);
+
+        unsigned int print_length = finger0_sample.data_shape.size() > 1 ? finger0_sample.data_shape[1] : finger0_sample.data_shape[0];
```

Comment:
It looks like there's some non-`ros` way to read the joint states. I _think_ this is what another ROS library would expose, but we don't want to do that through ROS here.